images/right-arrow.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="11" height="19" viewBox="0 0 11 19"><path fill="#ebebeb" d="M9.85 9.64L.92 1.14a.55.55 0 0 0-.76 0 .49.49 0 0 0 0 .73L8.71 10 .16 18.13a.5.5 0 0 0 0 .72.55.55 0 0 0 .76 0l8.92-8.5a.49.49 0 0 0 0-.71z"/></svg>
```

Comment:
Since you're using them directly in the HTML, why you're committing the resources too?